SAGESharp/SLB/Common.cs

```diff
@@ -396,4 +398,75 @@ private void SetByteValue(byte bytePosition, byte value)
             this.value = this.value.SetByte(bytePosition, value);
         }
     }
+
+    public sealed class Point3D : IEquatable<Point3D>
```

Comment:
Could we only use `Vector3D` instead of `Point3D`? It's the same information, and it's common to use a single 3D vector type to represent both points and directions.